2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ

```diff
@@ -97,20 +96,69 @@ Non sono definiti vincoli o limiti sulle memorie primaria e secondaria.
 
 ==== Operazioni
 
-// Dettagliare le operazioni, una volta confermate, secondo quanto descritto da 9.6.4.7
+//non ho molto chiaro cosa si richiedesse in questa sezione, sotto riporto lo standard
+//Specify the normal and special operations required by the user such as:
+// - the various modes of operations in the user organization*/
+Il sistema permette di attuare e visualizzare diverse operazioni di gestione del magazzino, in modo da semplificare l'interazione tra gestori e addetti.
+Il sistema deve permettere all'utente le seguenti operazioni:
```

Comment:
Riorganizzerei quanto scritto nell'elenco puntato, dividendolo in sezioni simili agli use case: Ergo qualcosa del genere:
- Magazzino:
      - creazione manuale del magazzino;
      - creazione mediante importazione del file SVG
      - modifica delle dimensioni del magazzino....
- Scaffali:
       - etc etc

Reply:
allora a questo punto non è meglio unire direttamente questo paragrafo e il 2.3.2 visto che sono praticamente quasi uguali?